Exiled.API/Features/Toys/Primitive.cs

```diff
@@ -121,6 +176,30 @@ public static Primitive Create(PrimitiveType primitiveType = PrimitiveType.Spher
 
             primitive.AdminToyBase.NetworkScale = primitive.AdminToyBase.transform.localScale;
             primitive.Base.NetworkPrimitiveType = primitiveType;
+            primitive.Color = color ?? Color.white;
```

Comment:
```suggestion
            primitive.Color = color ?? Color.gray;
```
Should match NW's default